lighthouse-core/report/report-generator.js

```diff
@@ -8,6 +8,10 @@
 const htmlReportAssets = require('./html/html-report-assets');
 
 class ReportGenerator {
+  static get Assets() {
```

Comment:
```suggestion
  static getAssets() {
```
any specific reason to have it as a getter?

Reply:
i thought it looked better when used: https://chromium-review.googlesource.com/c/chromium/src/+/1510732/5/third_party/blink/renderer/devtools/front_end/audits2/Audits2Panel.js

Reply:
I think they're usually just lying as an API--maybe justified if you're trying to match an expected interface--but we don't really have an established style on this front so to each their own :)

Should be lowercase `assets`, though